runtime/lua/vim/filetype.lua

```diff
@@ -770,6 +770,13 @@ local extension = {
   stm = function() vim.fn["dist#ft#FThtml"]() end,
   tcsh = function() vim.fn["dist#ft#SetFileTypeShell"]("tcsh") end,
   tex = function() vim.fn["dist#ft#FTtex"]() end,
+  --helpfiles match *.txt, but should have a modeline as last line
+  txt = function()
+    local lastline = vim.api.nvim_buf_get_lines(0, -2, -1, false)[1]
+    if not lastline:match("vim:.*ft=help") then
```

Comment:
```suggestion
    if not getline(bufnr,-1):match("vim:.*ft=help") then
```
(after rebase)

Reply:
Hey, what `getline` is this referring to? `vim.fn.getline` does not take a `bufnr` argument. Did I miss something?

Reply:
It's a local helper function defined near the top of the file.

Reply:
That's why I asked you to rebase ;)

Reply:
I went for `getbufline` :)